thirdeye/thirdeye-frontend/app/pods/self-serve/create-alert/controller.js

```diff
@@ -179,11 +181,12 @@ export default Controller.extend({
       };
 
       if (isCustomFilterPossible) {
-        const mttdVal = Number(customMttdChange).toFixed(2);
-        const severityThresholdVal = (Number(customPercentChange)/100).toFixed(2);
+        const mttdVal = Number.isNaN(Number(customMttdChange)) ? 0 : Number(customMttdChange).toFixed(2);
```

Comment:
Seeing this code repeated several times, Perhaps this could be abstracted into a utils function?

Reply:
Yes - I was thinking of doing that. TODO.